app/views/v2/select.scala.html

```diff
@@ -10,7 +10,7 @@
 
 @messageConstants = @{if(welshEnabled) WelshMessageConstants else EnglishMessageConstants}
```

Comment:
should the parameter be change to 'isWelsh'instead of 'welshEnabled' so it keeps a consistency throughout the other views?

Reply:
I've updated it for the select page now